packages/profile-sync-controller/src/controllers/authentication/AuthenticationController.ts

```diff
@@ -222,8 +222,6 @@ export default class AuthenticationController extends BaseController<
   }
 
   public performSignOut(): void {
-    this.#assertLoggedIn();
```

Comment:
We do not want to perform any assertions when signing out! We just need to reset/update state (see below state update)

Reply:
The intention for asserting the state was a good idea initially, but if there is some state mismatch, then they are unable to sign out!